yt_dlp/extractor/fptplay.py

```diff
@@ -0,0 +1,126 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import hashlib
+import time
+import urllib.parse
+
+from .common import InfoExtractor
+from ..utils import (
+    traverse_obj,
+    ExtractorError,
+    int_or_none,
+)
+
+
+class FptplayIE(InfoExtractor):
+    _VALID_URL = r'https?://fptplay\.vn/(?P<type>(?:xem-video))/(?P<slug>.*?)(?:\/(?P<episode>.*?)(?:\/|$)|$)'
+    _TESTS = [{
+        'url': 'https://fptplay.vn/xem-video/nhan-duyen-dai-nhan-xin-dung-buoc-621a123016f369ebbde55945',
+        'info_dict': {
+            'id': '621a123016f369ebbde55945',
+            'ext': 'mp4',
+            'title': 'Nhân Duyên Đại Nhân Xin Dừng Bước - Ms. Cupid In Love - tap-1',
+            'description': '23cf7d1ce0ade8e21e76ae482e6a8c6c',
+        },
+    }, {
+        'url': 'https://fptplay.vn/xem-video/ma-toi-la-dai-gia-61f3aa8a6b3b1d2e73c60eb5/tap-3',
+        'only_matching': True,
+    }, {
+        'url': 'https://fptplay.vn/xem-video/nha-co-chuyen-hi-alls-well-ends-well-1997-6218995f6af792ee370459f0',
+        'only_matching': True,
+    }]
+    _GEO_COUNTRIES = ['VN']
+    IE_NAME = 'fptplay'
+    IE_DESC = 'fptplay.vn'
+
+    def _real_extract(self, url):
+        type_url, slug, episode = self._match_valid_url(url).group('type', 'slug', 'episode')
+        if episode and int_or_none(episode.split('-')[-1]):
+            ep = int(int_or_none(episode.split('-')[-1]))
+        else:
+            ep = 0
+
+        video_id = slug.split('-')[-1]
+
+        webpage = self._download_webpage(url, video_id=video_id)
+
+        api = self.get_api_with_st_token(video_id, ep)
+
+        info = self._download_json(api, video_id=video_id)
```

Comment:
```suggestion
        info = self._download_json(self.get_api_with_st_token(video_id, ep), video_id=video_id)
```
May have to break into a new line.